mon-pix/app/routes/application.js

```diff
@@ -45,10 +48,11 @@ export default Route.extend(ApplicationRouteMixin, {
   // https://github.com/simplabs/ember-simple-auth/blob/a3d51d65b7d8e3a2e069c0af24aca2e12c7c3a95/addon/mixins/application-route-mixin.js#L132
   sessionInvalidated() {},
 
-  _setLocale() {
+  _setLocale(transition) {
```

Comment:
La responsabilité de `setLocale` est de changer de langue, aussi je pense qu'il est mieux de passer en paramètre `transition.to.queryParams.lang`.
```suggestion
  _setLocale(queryLocale) {
```